__tests__/app.test.js

```diff
@@ -127,3 +127,25 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("Status: 200 responds with an array of article objects containing author, title, article_id, topic, created_at, votes and comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toEqual(expect.any(Array));
+        expect(Object.keys(body.articles[0])).toEqual(
+          expect.arrayContaining([
```

Comment:
This should be an objectContaining, not an array! I'd also like it if you could wrap the logic you currently have in something that will iterate through and check it against each element in the array - not just [0]